modules/ROOT/pages/dw-core-functions-distinctby.adoc

```diff
@@ -11,6 +11,8 @@ endif::[]
 Iterates over an array and returns the unique elements in it.
 
 
+DataWeave uses the result of applying the provided lambda as the uniqueness criteria.
```

Comment:
TODO (DUKE): Add to Core.dwl

Reply:
also revised intro to "Iterates over the input and returns the unique elements in it."

Reply:
done